_config.yml

```diff
@@ -35,7 +35,7 @@ navigation:
 - text: Overview
   url: ''
   internal: true
-- text: ATOs
+- text: Lifecycle of a Launch
```

Comment:
This was based on feedback from a content designer I did a consult with. Maybe we can test it with users?